avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -1095,7 +1095,7 @@ function Set-UsageExamplesSection {
         $isUserDefinedType = $TemplateFileContent.parameters[$_].Keys -contains '$ref'
         $isNullable = $TemplateFileContent.parameters[$_]['nullable']
         $isNullableInRef = $TemplateFileContent.parameters[$_].Keys -contains '$ref' ? $TemplateFileContent.definitions[(Split-Path $TemplateFileContent.parameters[$_].'$ref' -Leaf)]['nullable'] : $false
-        (($hasNoDefaultValue -and -not $isUserDefinedType -and -not $isNullable) -or ($isUserDefinedType -and -not $isNullableInRef))
+        (($hasNoDefaultValue -and -not $isUserDefinedType -and -not $isNullable) -or ($isUserDefinedType -and -not $isNullableInRef -and -not $isNullable))
```

Comment:
Same comment of the orher PR, does this change impact any readme files for already published modules?

Reply:
It does not. I asked @ChrisSidebotham to run it an all scripts